bin/alluxio

```diff
@@ -258,6 +258,11 @@ function main {
   ;;
   "runTest")
     CLASS="alluxio.cli.TestRunner"
+    ALLUXIO_MASTER_HOSTNAME=$(${BIN}/alluxio getConf alluxio.master.hostname)
```

Comment:
Should this be 
`${ALLUXIO_MASTER_HOSTNAME:-$(${BIN}/alluxio getConf alluxio.master.hostname)}`?
Otherwise, defining `$ALLUXIO_MASTER_HOSTNAME` will not help but only having `alluxio.master.hostname` will work